### PR TITLE
SAK-46269 Podcasts: Ability to listen audio files.

### PR DESCRIPTION
**Pull request by mpellicer**
Super cheap and useful feature!

### CODE REVIEW

File: podcasts/podcasts-api/src/bundle/org/sakaiproject/api/podcasts/bundle/Messages.properties [hunk 1]
```diff
@@ -69,6 +73,9 @@ del_confirm=Are you sure you want to delete the following podcast?
 revise_title=Edit Podcast
 revise_directions=Update the form, then choose the appropriate button at the bottom.
 
+listen_title=Listen Podcast
+listen_instructions=Use the controls below to listen the podcast.
```
Comment (bjones86) [inline podcasts/podcasts-api/src/bundle/org/sakaiproject/api/podcasts/bundle/Messages.properties:77]:
```suggestion
listen_title=Listen to Podcast
listen_instructions=Use the controls below to listen to the podcast.
```

**Review by ottenhoff — APPROVED** [review]
Awesome!

**Comment by mpellicer** [comment]
> Awesome!

Less than two hours of work and a cool feature!

**Review by bjones86 — CHANGES_REQUESTED** [review]
Minor suggestion for English

**Review by bjones86 — APPROVED** [review]
Thanks!

**Pull request merged**